SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -2,20 +2,32 @@
 
 #include "Interop/SpatialPlayerSpawner.h"
 
-#include "Engine/Engine.h"
-#include "Engine/LocalPlayer.h"
-#include "Kismet/GameplayStatics.h"
-#include "TimerManager.h"
-
 #include "EngineClasses/SpatialNetDriver.h"
 #include "Interop/Connection/SpatialWorkerConnection.h"
 #include "Interop/SpatialReceiver.h"
+#include "LoadBalancing/AbstractLBStrategy.h"
+#include "Schema/ServerWorker.h"
+#include "Schema/UnrealObjectRef.h"
+#include "SpatialCommonTypes.h"
 #include "SpatialConstants.h"
+#include "SpatialGDKSettings.h"
 #include "Utils/SchemaUtils.h"
 
+#include "Engine/Engine.h"
+#include "Engine/LocalPlayer.h"
+#include "Containers/StringConv.h"
+#include "GameFramework/GameModeBase.h"
+#include "GameFramework/PlayerStart.h"
+#include "HAL/Platform.h"
+#include "Kismet/GameplayStatics.h"
+#include "TimerManager.h"
+#include "UObject/SoftObjectPath.h"
+
 #include <WorkerSDK/improbable/c_schema.h>
 #include <WorkerSDK/improbable/c_worker.h>
 
+#include <string>
```

Comment:
What's this used for?

Reply:
`strlen` for the macro replacement this guy made me use

Reply:
I hate that guy